.travis.yml

```diff
@@ -5,10 +5,13 @@ before_install:
   - docker build -t doc-build .
   - chmod +x docker-run.sh
   - chmod +x build.sh
+  - chmod +x travis-validation.sh
```

Comment:
You could change these to `chmod +x *.sh`, but if the whole point is to complain when the files are not `+x` shouldn't all of these be removed?